assignments/arrays.js

```diff
@@ -63,34 +63,131 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+const invSearch = (arr, invNum) => {
+   
+    for(i=0; i<arr.length; i++){
+        if(arr[i].id === invNum){
+            console.log(`Car ${arr[i].id} is a ${arr[i].car_year} ${arr[i].car_make} ${arr[i].car_model}`);
+        }
+    }
+};
+
+invSearch(inventory, 33);
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory.length -1;
+console.log(`The last car has id ${inventory[lastCar].id} and is a ${inventory[lastCar].car_year} ${inventory[lastCar].car_make} ${inventory[lastCar].car_model}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
+
+// const sortAlpha = function(arr){
+//     for(let i=0; i<arr.length; i++){
+//         carModels.push(arr[i].car_make);
+//     };
+//     retrun carModels; 
```

Comment:
I see you never actually invoke this, did you mean to leave `sortAlpha` in? there's a typo on line 92 which would have stopped a return anyway